spec/features/github_auth_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'feature_spec_helper'
+
+RSpec.describe 'Authenticate via github', feature: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.